src/main/java/com/example/demo/dao/impl/JdbcConnection.java

```diff
@@ -0,0 +1,134 @@
+        PreparedStatement statement = this.getDbConnection().prepareStatement(SqlRequestMessages.SQL_SELECT_ALL.getRequest());
+        ResultSet resultSet = statement.executeQuery();
+
+        while (resultSet.next()) {
+            Message message = new Message();
+            message.setId(resultSet.getLong("id"));
+            message.setSubject(resultSet.getString("subject"));
+            message.setEmailTo(resultSet.getString("emailTo"));
+            message.setMessage(resultSet.getString("message"));
+            message.setFutureSecond(resultSet.getLong("futureSecond"));
+            message.setCurrentTime(resultSet.getLong("currentTime"));
+            message.setStatus(Status.valueOf(resultSet.getString("status")));
+            messages.add(message);
+        }
+
+        transaction(statement);
+        return messages;
+    }
+
+    @Override
+    @SneakyThrows
+    public Message findById(Long id) {
+        Message message = new Message();
+        PreparedStatement statement = this.getDbConnection().prepareStatement(SqlRequestMessages.SQL_SELECT_MESSAGE_ID.getRequest());
+        statement.setLong(1, id);
+        ResultSet resultSet = statement.executeQuery();
+        if (resultSet.next()) {
+            message.setId(resultSet.getLong("id"));
+            message.setSubject(resultSet.getString("subject"));
+            message.setEmailTo(resultSet.getString("emailTo"));
+            message.setMessage(resultSet.getString("message"));
+            message.setFutureSecond(resultSet.getLong("futureSecond"));
+            message.setCurrentTime(resultSet.getLong("currentTime"));
+            message.setStatus(Status.valueOf(resultSet.getString("status")));
+        }
+
+        transaction(statement);
+        return message;
+
+    }
+
+    @Override
+    @SneakyThrows
+    public void updateEmailTimeById(Long id, Long time) {
+        PreparedStatement preparedStatement = this.getDbConnection().prepareStatement(SqlRequestMessages.SQL_UPDATE_TIME.getRequest());
+        preparedStatement.setLong(1, time);
+        preparedStatement.setLong(2, id);
+        preparedStatement.executeUpdate();
+        transaction(preparedStatement);
+    }
+
+    @Override
+    @SneakyThrows
+    public void updateEmailStatusById(Long id, Status status) {
+        PreparedStatement statement = this.getDbConnection().prepareStatement(SqlRequestMessages.SQL_UPDATE_STATUS.getRequest());
+        statement.setString(1, String.valueOf(status));
+        statement.setLong(2, id);
+        statement.executeUpdate();
+        transaction(statement);
+    }
+
+    @Override
+    @SneakyThrows
+    public void deleteById(Long id) {
+        PreparedStatement statement = this.getDbConnection().prepareStatement(SqlRequestMessages.SQL_DELETE_MESSAGE_ID.getRequest());
+        statement.setLong(1, id);
+        statement.execute();
+        transaction(statement);
+    }
+
+    private void transaction(PreparedStatement statement) throws SQLException {
```

Comment:
Give more meaningful name for this method.

Reply:
Done